components/automate-ui/src/app/pages/client-runs/client-runs.component.ts

```diff
@@ -580,7 +580,7 @@ export class ClientRunsComponent implements OnInit, OnDestroy {
 
   // If there is a page parameter in the URL the first page is not selected.
   isNotFirstPage(allUrlParameters: Chicklet[]): boolean {
-    return allUrlParameters.findIndex((parm: Chicklet) => parm.type === 'page') >= 0;
+    return allUrlParameters.some((parm: Chicklet) => parm.type === 'page');
```

Comment:
[nit] `parm`? interesting choice. But it's OK, of course.

Reply:
Oh, I meant to have 'param', but I changed it to have the full 'parameter'.